geopandas/geodataframe.py

```diff
@@ -512,6 +512,9 @@ def to_json(self, na="null", show_bbox=False, **kwargs):
             See below.
         show_bbox : bool, optional, default: False
             Include bbox (bounds) in the geojson
+        drop_id : bool, optional
+            Whether to retain the id. Default is False, but may want True
```

Comment:
It is not entirely clear that the `id` refers to the actual index of gdf. Can we expand the description a bit to make sure everyone knows that `drop_id` controls whether to preserve index as `id` property of json or not? Same applies to other docstrings below.

Reply:
Thanks, that's a good point.